packages/test/src/components/common/Button.tsx

```
@@ -1,19 +1,19 @@
-    );
+import React, { ReactNode } from 'react';
+
+interface ButtonProps {
+    children?: ReactNode;
+    onClick?: React.MouseEventHandler<HTMLDivElement>;
+    className?: string;
+    secondary?: boolean;
+    disabled?: boolean;
+}
+
+export const Button: React.FC<ButtonProps> = ({ children, className, secondary, disabled, ...props }) => {
```

ESLint

[error] 'className' is defined but never used.

(@typescript-eslint/no-unused-vars)